### ISSUE TITLE
Oversaturated when paused if OSD pops up

### ISSUE THREAD

**Issue by Quenser** [mpv-player/mpv#15130]
### mpv Information

```bash
mpv v0.39.0-113-g228f3860 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
 built on Oct 13 2024 00:08:22
libplacebo version: v7.349.0 (v7.349.0-15-g756220c-dirty)
FFmpeg version: N-117481-g02306cbfe
FFmpeg library versions:
   libavcodec      61.22.100
   libavdevice     61.4.100
   libavfilter     10.6.100
   libavformat     61.9.100
   libavutil       59.42.100
   libswresample   5.4.100
   libswscale      8.5.100
```

### Other Information

```text
- Windows version: Microsoft Windows [Version 10.0.22631.4317]
- GPU model, driver and version: RTX 3070 Driver ver. 560.94
- Source of mpv: mpv.io
- Introduced in version: cannot say
```

### Reproduction Steps

Exact same issue as in https://github.com/mpv-player/mpv/issues/11152
Screenshots in #11152 illustrate it perfectly.

Using a very similar TV.
Issue only occurs if watching content in HDR.

Content has to be started on the display with enabled HDR for the issue to occur. Does not occur if launched on SDR display and dragged over to HDR display.
Reproduceable with any type of content.  Independent of codec, HDR or SDR.

 --no-config fixes it. 
Issue occurs if manually commenting all configs out. 
Tried with no mpv.conf at all, issue still occurs.

Log is running with my default conf and on a different screen with HDR enabled because it kept launching on the main screen and not the one I am usually watching on.

### Expected Behavior

No Saturation when OSD shows up.

### Actual Behavior

Picture oversaturates to an insane degree when paused if I accidentally make the OSD pop up.
This is very troublesome for OLED displays as they react pretty badly to it even after only a few seconds.

### Log File

[output.txt](https://github.com/user-attachments/files/17447303/output.txt)

### I carefully read all instruction and confirm that I did the following:

- [X] I tested with the latest mpv version to validate that the issue is not already fixed.
- [X] I provided all required information including system and mpv version.
- [X] I produced the log file with the exact same set of files, parameters, and conditions used in "Reproduction Steps", with the addition of `--log-file=output.txt`.
- [X] I produced the log file while the behaviors described in "Actual Behavior" were actively observed.
- [X] I attached the full, untruncated log file.
- [X] I attached the backtrace in the case of a crash.

**Comment by quietvoid** [comment]
> Exact same issue as in https://github.com/mpv-player/mpv/issues/11152

So why did you make a new issue?

**Comment by guidocella** [comment]
Does `--gpu-shader-cache=no` fix it?

**Comment by llyyr** [comment]
Does `icc-profile-auto=no` fix it? `--no-config` also disables loading of any external files, which include icc profiles provided by Windows.

>  edit: I see it was suggested but no idea why.

The original issue didn't follow the template, was on mpv.net which is not mpv, and was on a much older version.

**Comment by Quenser** [comment]
> Does `--gpu-shader-cache=no` fix it?

No difference.

> Does `icc-profile-auto=no` fix it? `--no-config` also disables loading of any external files, which include icc profiles provided by Windows.

No difference either.

**Comment by guidocella** [comment]
`--icc-cache=no` should fix it then.

**Comment by Quenser** [comment]
> `--icc-cache=no` should fix it then.

Unfortunately, issue still persists. 
Also funnily enough if I open it on an affected monitor and drag it over to another (in SDR mode) it persists over there too.

What I find really weird is how the OSD seems to trigger it. If I pause the video and don't pop the OSD up there are no issues.

**Comment by Doofussy2** [comment]
This looks like an issue i reported a long time ago.

https://github.com/mpv-player/mpv/issues/7406

And using vo=gpu I can also see this issue, but not with gpu-next.

**Comment by aloola18** [comment]
this happened to me on Windows too, with Windows HDR turned on + mpv with vo=gpu
Windows HDR turned off + mpv with vo=gpu => fine
Windows HDR on + mpv with vo=gpu-next => fine

**Comment by Doofussy2** [comment]
If you configure the --target-trc and --target-prim, it should be correct

**Comment by kasper93** [comment]
After #16512 everything should be configured correctly automatically. **For HDR remember to use `vo=gpu-next`**, there are no plans to improve `vo=gpu` which will blink, because of incorrect surface format and inability to send HDR metadata.

Let me know if there is anything more we can do for you.

**Comment by pierino971** [comment]
Hi guys, I'm trying to understand why for me the option vo=gpu-next does not wok, I have a 9070xt as a GPU, I also attach my configs so that maybe you can tell me what I'm setting wrong. Thanks in advance 

[Configs.txt](https://github.com/user-attachments/files/21230586/Configs.txt)

**Comment by Doofussy2** [comment]
> Hi guys, I'm trying to understand why for me the option vo=gpu-next does not wok, I have a 9070xt as a GPU, I also attach my configs so that maybe you can tell me what I'm setting wrong. Thanks in advance
> 
> [Configs.txt](https://github.com/user-attachments/files/21230586/Configs.txt)

Try using d3d11 instead of vulkan

**Comment by kasper93** [comment]
> Hi guys, I'm trying to understand why for me the option vo=gpu-next does not wok, I have a 9070xt as a GPU, I also attach my configs so that maybe you can tell me what I'm setting wrong. Thanks in advance
> 
> [Configs.txt](https://github.com/user-attachments/files/21230586/Configs.txt)

Post log, not config.

**Comment by pierino971** [comment]
Oh my bad, here it is: 
[output.txt](https://github.com/user-attachments/files/21337545/output.txt)

**Comment by kasper93** [comment]
> Oh my bad, here it is: [output.txt](https://github.com/user-attachments/files/21337545/output.txt)

when you try to play someting

**Comment by pierino971** [comment]
Sorry for the late response but work got me busy, here it is:
This one is done while playing the video:
[output2.txt](https://github.com/user-attachments/files/21383152/output2.txt)
This One is done while the video is paused:
[output.txt](https://github.com/user-attachments/files/21383151/output.txt)

**Comment by kasper93** [comment]
> mpv v0.39.0-1025-g6c4218252

Update your mpv first.

**Comment by pierino971** [comment]
I updated mpv and now everything works fine, thanks (I didn't even think that was not updated and THAT was the problem
